corehq/change_ownership.py

```diff
@@ -0,0 +1,202 @@
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.models import CommCareCase
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.es import CaseSearchES
+from corehq.apps.es.cases import case_type
+import os
+import time
+import math
+
+from casexml.apps.case.mock import CaseBlock
+from corehq.apps.hqcase.utils import submit_case_blocks
+
+# DOMAIN = 'alafiacomm'
+DOMAIN = 'alafiacomm-prod'
+
+success_case_log_file_path = os.path.expanduser('~/script_success.log')
```

Comment:
nit: I prefer adding timestamps to filenames so in case of re-run they don't conflict.

Reply:
Good idea! Given the scope of how many cases we'll be dealing with, do you think it will be fine to have everything inside a single file, or should we chunk our logging?

Reply:
This is a great suggestion, and I think one that is easily achievable with how the script has been set up now. We can simply store the data for each case type in a separate DB table for processing.

I've updated the script to make `CaseUpdater` only handle a single case type now ([559dfad](https://github.com/dimagi/commcare-hq/pull/34588/commits/559dfadba25c08c59016e3c54a066ca2cc863bdc)).